examples/acados_python/pendulum_on_cart/ocp/example_sqp_rti_loop.py

```diff
@@ -145,6 +145,9 @@ def main():
     cost = ocp_solver.get_cost()
     print("cost function value of solution = ", cost)
 
+    # test getter
```

Comment:
Same for constraints_get? Can we add more tests for all the new getters?

Reply:
will be tested by evaluator :smile:, cf https://github.com/acados/acados/pull/1402